W-17030250: Clarify backup&restore rtfctl secure properties management (#1388)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/manage-backup-restore.adoc

=== Changes Made After Performing a Backup Are Not Restored

> All changes performed on the control plane side do not need a new backup to be taken into account since those are taken from the platform at the time of restoration.

Reviewer: ```suggestion
All changes performed on the control plane side don't need a new backup to be taken into account because those are taken from the platform at the time of restoration.
```

> Any change done on the cluster side are not part of the restoration and they need to be reapplied. These changes include:

Reviewer: ```suggestion
Any changes done on the cluster side aren't part of the restoration and must be reapplied. These changes include:
```
